src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/register_app_interface_request.h

```diff
@@ -215,6 +215,15 @@ class RegisterAppInterfaceRequest
    */
   bool IsApplicationSwitched();
 
+   /**
+   * @brief ApplicationWithSameNameAndID checks that the application which 
+   * sends `RegeisterAppInterface` request doesn't have same app id and same
+   * app name, and check that the doesn't already registered application.
```

Comment:
@ValeriiMalkov "checks that there are no already registered applications with the same app id and app name as for application which is sending `RegeisterAppInterface` request "